subsys/bluetooth/mesh/subnet.h

```diff
@@ -71,7 +75,9 @@ struct bt_mesh_subnet {
 		uint8_t identity[16];  /* IdentityKey */
 	#endif
 		uint8_t beacon[16];      /* BeaconKey */
+#if defined(CONFIG_BT_MESH_V1d1)
```

Comment:
Should it be `CONFIG_BT_MESH_PRIV_BEACONS` instead?

Reply:
Mesh-1.1 node shall be able to process private beacons. `CONFIG_BT_MESH_PRIV_BEACONS` only enables transmission of the private beacons.